.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: build
+
+on:
+    push:
+        branches: ["main"]
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        permissions: # required by aws-actions/configure-aws-credentials
+            id-token: write
+            contents: read
+
+        steps:
+            # Seed the build number with last number from TeamCity.
+            # This env var is used by the JS, and SBT builds, and guardian/actions-riff-raff.
+            # Set the value early, rather than `buildNumberOffset` in guardian/actions-riff-raff, to ensure each usage has the same number.
+            # For some reason, it's not possible to mutate GITHUB_RUN_NUMBER, so set BUILD_NUMBER instead.
+            - name: Set BUILD_NUMBER environment variable
+              run: |
+                  LAST_TEAMCITY_BUILD=1603
+                  echo "BUILD_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))" >> $GITHUB_ENV
+            - uses: actions/checkout@v4
+            - uses: actions/setup-java@v3
+              with:
+                  distribution: "corretto"
+                  java-version: "8"
+                  cache: "sbt"
+
+            - name: AWS Auth
+              uses: aws-actions/configure-aws-credentials@v4
+              with:
+                  role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+                  aws-region: eu-west-1
+
+            - name: Install Node
+              uses: actions/setup-node@v3
+              with:
+                node-version-file: '.nvmrc'
+
+            - name: Install dependencies, build & run tests
+              run: |
+                  set -e
+                  yarn install --frozen-lockfile
+                  yarn build
+
+            - name: Build and Test project
+              run: |
+                  sbt clean compile test Debian/packageBin
+
+            - name: Notification Lambda
+              run: |
+                  sbt "project notification" clean compile Universal/packageBin
+
+            - uses: guardian/actions-riff-raff@v2
+              with:
+                  projectName: Editorial Tools::Workflow::Workflow Frontend
+                  buildNumber: ${{ env.BUILD_NUMBER }}
+                  configPath: conf/riff-raff.yaml
+                  contentDirectories: |
+                      workflow-frontend:
+                        - target/workflow-frontend_0.1_all.deb
+                      workflow-frontend-fluentbit:
+                        - fluentbit/parsers.conf
+                        - fluentbit/td-agent-bit.conf
```

Comment:
not for this PR, but looking forward to when we move to `devx-logs` (https://trello.com/c/P8JrJLHR/1345-use-devx-logs-for-workflow-frontend)